tensorflow-to-larod-cv25/README.md

```diff
@@ -110,8 +110,7 @@ The following instructions can be executed to simply run the example. Each step
 
 4. Compile the model using the [Ambarella toolchain](https://www.ambarella.com/technology/#cvflow)
 
-   >NOTE
-   *> It is required to be an Ambarella partner to access Ambarella tools. If you are a partner, visit [this](https://customer.ambarella.com/ng/pre-login) webpage to access the tools.*
+   > NOTE > It is required to be an Ambarella partner to access Ambarella tools. Visit [this](https://customer.ambarella.com/ng/login?refid=EAAAALA9%2fIfpWDCn53oQJDd5FKfzsrI0fWXYseVTpgnkJHV1) webpage to access the tools. If you are not an Ambarella partner, you can get login credentials by registering in the same link.
```

Comment:
I get back to this link
```suggestion
   > NOTE > It is required to be an Ambarella partner to access Ambarella tools. Visit [this](https://customer.ambarella.com/ng/login) webpage to access the tools. If you are not an Ambarella partner, you can get login credentials by registering in the same link.
```

Reply:
Actually, If you use my link, there is a "register" button that doesn't appear when using your link.

Reply:
OK!